qiskit/compiler/assemble.py

```diff
@@ -247,8 +248,12 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
     schedule_los = [lo_config if isinstance(lo_config, LoConfig) else LoConfig(lo_config)
                     for lo_config in schedule_los]
 
-    qubit_lo_freq = qubit_lo_freq or getattr(backend_default, 'qubit_freq_est', None)
-    meas_lo_freq = meas_lo_freq or getattr(backend_default, 'meas_freq_est', None)
+    warnings.filterwarnings("ignore")
```

Comment:
why?

Reply:
yeah, the `qubit_freq_est` raises a warning because its behavior has changed. I don't want this warning showing when users are using the assembler, since it isn't their concern (the assembler is working in sync with the pulse defaults changes). I mentioned you in another thread -- I would be interested in discussing alternative migration paths for `qubit_freq_est` and `meas_freq_est`